xrdp/xrdp_mm.c

```diff
@@ -1152,10 +1050,19 @@ dynamic_monitor_data(intptr_t id, int chan_id, char *data, int bytes)
     struct xrdp_process *pro;
     struct xrdp_wm *wm;
     int monitor_layout_size;
+    struct dynamic_monitor_description *dynamic_description;
 
     LOG_DEVEL(LOG_LEVEL_TRACE, "dynamic_monitor_data:");
     pro = (struct xrdp_process *) id;
     wm = pro->wm;
+
+    if (wm->client_info->suppress_output == 1)
```

Comment:
Here is the documentation on the suppress output functionality. https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/79202daa-76b8-4a09-ac67-fc3cae7d895f

Essentially: The client can send a message to the server to prevent any display output updates from the server. Though thinking on this more, it only says that display output should be suppressed, it's reasonable to assume that we should still process resizing messages during this.

I'll remove.